docs/source/redhat-opens-install.rst

```diff
@@ -312,17 +312,10 @@ On each control node, perform the following steps:
    OpenStack packages and to ``dnsmasq``.  (OpenStack updates are not needed
    for Liberty.)
 
-3. Edit the ``/etc/neutron/plugins/ml2/ml2_conf.ini`` file.  In the `[ml2]`
-   section:
+3. Edit the ``/etc/neutron/neutron.conf`` file.  In the `[DEFAULT]` section:
 
-   -  Find the ``type_drivers`` setting and change it to read
-      ``type_drivers = local, flat``.
-   -  Find the ``tenant_network_types`` setting and change it to read
-      ``tenant_network_types = local``.
-   -  Find the ``mechanism_drivers`` setting and change it to read
-      ``mechanism_drivers = calico``.
-
-4. Edit the ``/etc/neutron/neutron.conf`` file.  In the `[DEFAULT]` section:
+   -  Find the line beginning with ``core_plugin``, and change it to read
+      ``core_plugin = calico.openstack.core_calico.CalicoCorePlugin``.
```

Comment:
@nbartos I think I need a steer from @lxpollitt and @neiljerram on whether it's OK to do away with the ML2-only instructions.  I'd be inclined to document both with the first option being "For a fully featured Calico deplyment with floating IP support..." and the other being "If you really know what you're doing and you're willing to live without floating IP support..."

Replacing the CorePlugin is probably fine for most people (and it's what allows us to hook the FIP calls) but it does mean that your network ends up being Calico-only.